### PR TITLE
[12.0][FIX] *: lift obsolete many2one constraints

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
I only checked this three modules, as they are very important.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why not removing FKs also on obsolete models?

**Comment by MiquelRForgeFlow** [comment]
The ones in `account` you mean? I was too lazy haha

**Comment by pedrobaeza** [comment]
Yeah, but the same problem that you have faced with fields may arise due to this.

**Comment by MiquelRForgeFlow** [comment]
Ready!

File: addons/account/migrations/12.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -130,6 +130,26 @@ def drop_obsolete_constraint_wizard_multi_charts_accounts(env):
     has been removed in V12.0
     """
     openupgrade.remove_tables_fks(env.cr, ['wizard_multi_charts_accounts'])
+    openupgrade.remove_tables_fks(env.cr, ['account_aged_trial_balance'])
```
Comment (pedrobaeza) [inline addons/account/migrations/12.0.1.1/pre-migration.py]: Why not passing all tables in one call?

Comment (MiquelRForgeFlow) [reply addons/account/migrations/12.0.1.1/pre-migration.py]: done

**Pull request merged**